### PR TITLE
Add initial migration script to include TCR tables & custom query tables

### LINKED ISSUE #2437: Migration #1: Port TCR SQLite DB to Postgres
Write the initial Flyway migrations to port the TCR SQLite DB to Postgres.

### PR DESCRIPTION
**Pull request by m-goggins**
# PULL REQUEST

## Summary
Adds the initial migration SQL script for Flyway to create the TCR tables and the custom query tables. This should allow Brandon to implement Flyway.

## Related Issue
Fixes #2437 

## Acceptance Criteria
Please copy the acceptance criteria from your ticket and paste it here for your reviewer(s)

## Additional Information
Anything else the review team should know?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/tefca-viewer/migrations/V01_01__tcr_custom_query_schema.sql [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    id TEXT PRIMARY KEY,
+    valueset_id TEXT,
+    concept_id TEXT,
+    FOREIGN KEY (valueset_id) REFERENCES valuesets(id),
+    FOREIGN KEY (concept_id) REFERENCES concepts(id)
+);
+
+CREATE TABLE IF NOT EXISTS icd_crosswalk (
+    id TEXT PRIMARY KEY,
+    icd10_code TEXT,
```
Comment (bamader) [inline containers/tefca-viewer/migrations/V01_01__tcr_custom_query_schema.sql:48]: In your local spinup demo, I'm assuming we wound up being okay not having the `FOREIGN KEY()` referencing `gem_formatted_code` on this? And also not needing that additional crosswalk mapping table that we talked about?

**Review by bamader — APPROVED** [review]
LGTM if we don't need the `FOREIGN KEY` tag!

**Review by DanPaseltiner — CHANGES_REQUESTED** [review]
Can we create indexes on all primary and foreign keys? We learned previously that PG doesn't do this automatically and join performance will suffer without them.

File: containers/tefca-viewer/migrations/V01_01__tcr_custom_query_schema.sql [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    icd9_code TEXT,
+    match_flags TEXT);
+
+
+CREATE TABLE IF NOT EXISTS query (
```
Comment (DanPaseltiner) [inline containers/tefca-viewer/migrations/V01_01__tcr_custom_query_schema.sql:53]: These don't look like the right columns for the query table

Comment (m-goggins) [reply containers/tefca-viewer/migrations/V01_01__tcr_custom_query_schema.sql:53]: Ah whoops, left over copy+paste

**Comment by m-goggins** [comment]
Good point, @DanPaseltiner! I added all the indexes for primary and foreign keys

**Review by robertandremitchell — APPROVED** [review]
lgtm!

**Review by DanPaseltiner — APPROVED** [review]

**Pull request merged**